### DISCUSSION TITLE
[Football API] Determine starters in a given week

### DISCUSSION THREAD

**Discussion by sebcl** [cwendt94/espn-api#302] [Q&A]
Per the BoxScore class, I can see the home/away lineups. From there I can figure out week breakdowns for a given team and the players associated with that team. 

Unfortunately, I can't seem to figure out which players were actually started. Any ideas here?

My only thought here is to compare player actual values and figure out the combination that gets to a team's output for that given week but that seems a bit much.

**Comment by cwendt94** [comment]
You can use the [Box Player](https://github.com/cwendt94/espn-api/wiki/Box-Player-Class) `slot_position` to filter out non starters. Just need to filter out `BE` and `IR` slot positions.
